doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -261,7 +261,7 @@ HW profiles are defined in the chapters 5.3 and 5.4 of the reference model docum
 #### 3.4.2.2. Network
 The recommended network architecture is spine and leaf topology; however, for small sites, a legacy topology (access/aggregation switches) can be set up. 
 
```

Comment:
the topology is approved but can we remove this "a legacy topology (access/aggregation switches) can be set up"

Reply:
@rabi-abdel ?

Reply:
@ASawwaf this needs to be fixed but since this PR addressing a different issue, i suggest creating an issue to address this particular point.

Reply:
https://github.com/cntt-n/CNTT/issues/1311